packages/marshal/src/marshal.js

```diff
@@ -180,20 +187,23 @@ function isPassByCopyError(val) {
     stack: _optStackDesc,
     ...restDescs
   } = getOwnPropertyDescriptors(val);
-  const restKeys = ownKeys(restDescs);
-  assert(
-    restKeys.length === 0,
-    X`Unexpected own properties in error: ${q(restKeys)}`,
-    TypeError,
-  );
-  if (mDesc) {
-    assert.typeof(mDesc.value, 'string', X`Malformed error object: ${val}`);
-    assert(
-      !mDesc.enumerable,
-      X`An error's .message must not be enumerable`,
-      TypeError,
+  if (ownKeys(restDescs).length >= 1) {
+    assert.note(
+      val,
+      X`Pass-by-copy error has extra unpassed properties ${restDescs}`,
     );
   }
+  if (mDesc) {
+    if (typeof mDesc.value !== 'string') {
+      assert.note(
+        val,
+        X`Error message property ${mDesc} must be a string-valued data property.`,
+      );
+    }
+    if (mDesc.enumerable) {
+      assert.note(val, X`Error message property ${mDesc} not be enumerable`);
```

Comment:
If I'm reading this right: 
```suggestion
      assert.note(val, X`Error message property ${mDesc} must not be enumerable`);
```

Reply:
You read that right. Thanks!